lib/ui/window.dart

```diff
@@ -210,7 +210,19 @@ class Window {
   /// [physicalSize], or [padding] values change, for example when the device is
   /// rotated or when the application is resized (e.g. when showing applications
   /// side-by-side on Android).
-  VoidCallback onMetricsChanged;
+  ///
+  /// The framework invokes this callback in the same zone in which the
+  /// callback was set.
+  VoidCallback get onMetricsChanged => _onMetricsChanged;
+
```

Comment:
nit: we usually don't bother with these blank lines, we consider the getter/field/setter group to be one piece of code.